MoltenVK/MoltenVK/Layers/MVKExtensions.def

```diff
@@ -63,6 +63,7 @@ MVK_EXTENSION(KHR_external_memory_capabilities,       KHR_EXTERNAL_MEMORY_CAPABI
 MVK_EXTENSION(KHR_external_semaphore,                 KHR_EXTERNAL_SEMAPHORE,                 DEVICE,   10.11,  8.0,  1.0)
 MVK_EXTENSION(KHR_external_semaphore_capabilities,    KHR_EXTERNAL_SEMAPHORE_CAPABILITIES,    INSTANCE, 10.11,  8.0,  1.0)
 MVK_EXTENSION(KHR_fragment_shader_barycentric,        KHR_FRAGMENT_SHADER_BARYCENTRIC,        DEVICE,   10.15, 14.0,  1.0)
+MVK_EXTENSION(KHR_format_feature_flags2,              KHR_FORMAT_FEATURE_FLAGS_2,             DEVICE,   10.11,  8.0,  1.0)
```

Comment:
Also update list in `MoltenVK_Runtime_UserGuide.md` (by ext type then alpha order)

Reply:
`Whats_New.md` too?

Reply:
Either is fine.

For me, I normally just add fixes to `Whats_New.md` when I do them. But you are correct...I do a PR sweep into `Whats_New.md` at release time. 

I'll be doing that later this week, so no need for you to update it now if you haven't done so yet.